config/rbac/spark-application-rbac.yaml

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: spark-operator-spark
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: default
+  name: spark-role
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["*"]
+- apiGroups: [""]
+  resources: ["services"]
+  verbs: ["*"]
```

Comment:
I think these permissions are not enough, ref: https://github.com/kubeflow/spark-operator/blob/f1b8d468f6b496bfd9577d056474155418878312/charts/spark-operator-chart/templates/spark/rbac.yaml#L33-L49.

Reply:
@ChenYi015 Thank you for your review.
Of course I will update.

Do you want me to do the following?
**1. update spark-application-rbac.yaml**
Add configmaps and persistentvolumeclaims and modify verb as mentioned in "spark-application-rbac.yaml" file.
**2. update [docs](https://www.kubeflow.org/docs/components/spark-operator/getting-started/#about-the-service-account-for-driver-pods)**
If config/rbac/spark-application-rbac.yaml is added, modify it to the corresponding path.
as is : spark-operator/manifest/spark-application-rbac/spark-application-rbac.yaml
to be : config/rbac/spark-application-rbac.yaml

Did I understand it correctly?

Reply:
> Do you want me to do the following?
1. update spark-application-rbac.yaml
Add configmaps and persistentvolumeclaims and modify verb as mentioned in "spark-application-rbac.yaml" file.
2. update [docs](https://www.kubeflow.org/docs/components/spark-operator/getting-started/#about-the-service-account-for-driver-pods)
If config/rbac/spark-application-rbac.yaml is added, modify it to the corresponding path.
as is : spark-operator/manifest/spark-application-rbac/spark-application-rbac.yaml
to be : config/rbac/spark-application-rbac.yaml

@ha2hi  Sorry for the late reply. Yes, please update both the yaml file and docs. And it would be better if you can rebase commits of this PR onto the master branch.

Reply:
@ChenYi015 it's okay. I modified "spark-application-rbac.yaml". 
Could you please Rereview it?

The docs will be updated when the yaml file is added.